manage.py

```diff
@@ -251,6 +249,48 @@ def demo_db():
     for i in range(5):
         add_passover(timestamp=now + timedelta(minutes=i*10))
 
+    print("Database has been seeded with demo data.")
+
+
+@cli.command('import_commands')
+def import_commands():
+    """Imports commands from the CommandDocs.txt file in ex2_ground_station_software,
+    and adds them to the database.
+    """
+
+    filepath = 'ex2_ground_station_software/CommandDocs.txt'
+
+    if not os.path.exists(filepath):
+        print(f'Couldn\'t find list of commands at {filepath}. Seeding database with no commands.')
+        return False
+
+    # first regenerate CommandDocs.txt
+    print('Regenerating commands.')
+    subprocess.run(['./regenerate_commands.sh'])
```

Comment:
Still some bugs to fix before this can be merged, I'm currently getting a FileNotFound error here.